### PR TITLE
Unittest for pinned_post.dart

### PR DESCRIPTION
**Pull request by nidhin29**
This change included the Unittest for pinned post

Issue Number: https://github.com/PalisadoesFoundation/talawa/issues/2161

Fixed https://github.com/PalisadoesFoundation/talawa/issues/2161

Did you add tests for your changes?

Yes

Summary

It is unittest for pinned_post.

Does this PR introduce a breaking change?

No, this does not introduce any major changes to the codebase.

Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?

Yes, i have followed the guidelines.

### CODE REVIEW

**Comment by nidhin29** [comment]
@noman2002 The pinned post are manually passed so is it possible to check the error cases

**Comment by nidhin29** [comment]
@noman2002  Sir,I have written tests for other lines if it is possible to test the error case pls tell me and I will work on it

File: lib/widgets/pinned_post.dart [hunk 1]
```diff
@@ -100,37 +101,43 @@ class PinnedPost extends StatelessWidget {
...
+                              imageUrl:
+                                  (pinnedPosts[index]['imageUrl'] ?? '').isEmpty
+                                      ? 'placeHolderUrl'
+                                      : pinnedPosts[index]['imageUrl']!,
+                              errorWidget: (context, url, error) {
+                                return const CircularProgressIndicator();
+                              },
+                              height: SizeConfig.screenHeight! * 0.15,
+                              fit: BoxFit.cover,
+                            ),
```
Comment (noman2002) [inline lib/widgets/pinned_post.dart:127]: Please write the test for this error widget.

**Review by noman2002 — CHANGES_REQUESTED** [review]

**Comment by noman2002** [comment]
> tests for other lines if it

Yes, Please write the test for error widget. You can mock the situation by giving incorrect image url.

**Pull request closed without merging**